nsca_client/demo/demo_data.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record id="demo_nsca_server" model="nsca.server">
```

Comment:
Would it make sense to add a default server as regular data, so that additional modules containing checks can use its XML id?